crates/red_knot_python_semantic/resources/mdtest/snapshots/not.md_-_Unary_not_-_Object_that_implements_`__bool__`_incorrectly.snap

```diff
@@ -22,12 +22,13 @@ mdtest path: crates/red_knot_python_semantic/resources/mdtest/unary/not.md
 # Diagnostics
 
 ```
-error: lint:unsupported-bool-conversion: Boolean conversion is unsupported for type `NotBoolable`; its `__bool__` method isn't callable
+error: lint:unsupported-bool-conversion: Boolean conversion is unsupported for type `NotBoolable`
  --> /src/mdtest_snippet.py:5:1
   |
 4 | # error: [unsupported-bool-conversion]
 5 | not NotBoolable()
   | ^^^^^^^^^^^^^^^^^
   |
+info: `__bool__` must be callable
```

Comment:
In all of these cases, it would be ideal to have an annotation pointing at the not-callable `__bool__` in question (doesn't need to happen in this PR)

Reply:
Yeah I called this out in the code. I actually tried to do it, but it didn't seem straight-forward. (And I mean that in the sense that it is maybe actually not straight-forward, or more likely, only not straight-forward to _me_ haha.)